REST/PowerShell/Runbooks/CopyRunbooks.ps1

```diff
@@ -153,6 +160,22 @@ foreach ($sourceRunbook in $sourceProjectRunbooks)
             {
                 $action.WorkerPoolId = $destinationWorkerPool.Id
             }
+
+            if ($null -ne $action.Properties.'Octopus.Action.Template.Id')
+            {
+                $sourceActionTemplate = $sourceActionTemplates | Where-Object {$_.Id -eq $action.Properties.'Octopus.Action.Template.Id'}
+
+                # Check destination to see if that template was installed
+                $destinationActionTemplate = $destinationActionTemplates | Where-Object {$_.CommunityActionTemplateId -eq $sourceActionTemplate.CommunityActionTemplateId}

```

Comment:
Will checking the destination communityactiontemplateid reliably against the destination templateid reliably work? Surely the CommunityActionTemplateId isn't consistent (or at least guaranteed to be) across instances? 

I think it would be better to do it based on the names matching and the CommunityActionTemplateId being populated (eg not null)?

I checked the samples-admin process to manage community steps and [this is what I found](https://github.com/OctopusSamples/IaC/blob/master/octopus-samples-instances/api-scripts/manage-community-step-templates.ps1#L97) which also looks like it does comparisons based on name?